index.html

```diff
@@ -0,0 +1,128 @@
+        <h2 class="program-date">2022.06.15(FRI)</h2>
+        <p class="program-venue">@ Bangabandhu Internation Conference Center, Dhaka</p>
+      </div>
+    </section>
+
+    <section id="events" class="events">
+      <h2 class="programAnnounce">Main Program</h2>
+      <hr>
+      <div class="events-container">
+        <article class="event-box">
+          <img src="image/icons/lecture.png" alt="lecture">
+          <h3 class="event-title">Lecture</h3>
+          <p class="event-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia, eligendi?</p>
+        </article>
+
+        <article class="event-box">
+          <img src="image/icons/exhibition.png" alt="exibition">
+          <h3 class="event-title">Exibition</h3>
+          <p class="event-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione, sunt.</p>
+        </article>
+
+        <article class="event-box">
+          <img src="image/icons/forum.png" alt="forum">
+          <h3 class="event-title">Forum</h3>
+          <p class="event-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora, commodi!</p>
+        </article>
+
+        <article class="event-box">
+          <img src="image/icons/workshop.png" alt="workshop">
+          <h3 class="event-title">Workshop</h3>
+          <p class="event-details">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sunt!</p>
+        </article>
+
+        <article class="event-box">
+          <img src="image/icons/ignite.png" alt="ignit">
+          <h3 class="event-title">SCUT Ignit</h3>
+          <p class="event-details">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dolore, voluptas.</p>
+        </article>
+        </div>
+      <button type="button" class="programsButton">See Event Details</button>
+
+    </section>
+
+  <!-- speakers section -->
+  <section id="speakers" class="speakers-sec">
+    <h2 class="performers_header">Speakers</h2>
+    <hr />
+    <div class="speakers-group"></div>
+  </section>
+</main>
+
+<footer id="footer">
+  <section class="sponsor">
+    <h2 class="sponsor-head">Sponsors</h2>
+    <hr>
+    <div class="sponsor-images-container">
+      <img class="sponsor-img" src="image/images/partners/airbnb.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/navar.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/daumk.svg" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/Google.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/mozilla.png" alt="sponsor-img">
+    </div>
```

Comment:
- [ ] Please, consider giving different alternative text for images - Google for Google, Airbnb for Airbnb - it will be easy to recognize images if they will not load on the page 👍 

> Please, consider the same for About page 💯

Reply:
Alternative text in the `img` tag has been changed as required in both index.html and about.html

Reply:
resolved 👍